programs/stake/src/stake_state.rs

```diff
@@ -969,6 +1007,17 @@ pub fn calculate_points(
     }
 }
 
+// utility function, used by Split
+fn calculate_split_rent_exempt_reserve(
+    source_meta: &Meta,
+    source_data_len: u64,
+    split_data_len: u64,
+) -> u64 {
+    let lamports_per_byte_year =
+        source_meta.rent_exempt_reserve / (source_data_len + ACCOUNT_STORAGE_OVERHEAD);
+    lamports_per_byte_year * (split_data_len + ACCOUNT_STORAGE_OVERHEAD)
```

Comment:
Also, let's cross-ref this out-of-wild rent calculation was unavoidable at `Rent` in rent.rs. :)

Reply:
Where would you like to see that comment in rent.rs?